packages/runtime/container-runtime/src/summaryManager.ts

```diff
@@ -57,7 +75,7 @@ class QuorumHeap {
         }
     }
 
-    public removeClient(clientId: string) {
+    private removeClient(clientId: string) {
```

Comment:
i think we should emit heap change from within these methods, near where we actually change the heap